docs/mkdocs/documentation/install.md

```diff
@@ -1,7 +1,133 @@
+
+If you want to install Kernel Module Management programmatically, you can use the resources below to create the
+`Namespace`, `OperatorGroup` and `Subscription` resources.
+
+```yaml
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-kmm
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: kernel-module-management
+  namespace: openshift-kmm
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: kernel-module-management
+  namespace: openshift-kmm
+spec:
+  channel: release-1.0
+  installPlanApproval: Automatic
+  name: kernel-module-management
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: kernel-module-management.v1.0.0
+```
+
 ## Using `kubectl`
 
+The command below installs the bleeding edge version of KMM.
+To install from a release branch, replace `main` with the name of the release branch, e.g. `release-1.0`.
+
 ```shell
 kubectl apply -k https://github.com/rh-ecosystem-edge/kernel-module-management/config/default
-```
\ No newline at end of file
+```
+
+## OpenShift versions below 4.12
+
+!!! note "KMM is supported on OpenShift 4.12 and above."
+
+Installing KMM on OpenShift 4.11 does not require specific steps.
+
+For versions 4.10 and below, some RBAC adjustments need to be made before you create the `OperatorGroup` and the
+`Subscription` objects.  
+Because KMM is designed to work with OpenShift's 4.12 security features, you need to create a new
+`SecurityContextConstraint` object and to bind it to the operator's `ServiceAccount`.
+Those steps need to happen after you have created the `Namespace`, but before you create the `OperatorGroup`, install
+through the OpenShift console or run `kubectl apply`.
+
+<details>
+<summary>Additional RBAC for OpenShift 4.10</summary>
+
+```yaml
+---
+allowHostDirVolumePlugin: false
+allowHostIPC: false
+allowHostNetwork: false
+allowHostPID: false
+allowHostPorts: false
+allowPrivilegeEscalation: false
+allowPrivilegedContainer: false
+allowedCapabilities:
+  - NET_BIND_SERVICE
+apiVersion: security.openshift.io/v1
+defaultAddCapabilities: null
+fsGroup:
+  type: MustRunAs
+groups: []
+kind: SecurityContextConstraints
+metadata:
+  name: restricted-v2
+priority: null
+readOnlyRootFilesystem: false
+requiredDropCapabilities:
+  - ALL
+runAsUser:
+  type: MustRunAsRange
+seLinuxContext:
+  type: MustRunAs
+seccompProfiles:
+  - runtime/default
+supplementalGroups:
+  type: RunAsAny
+users: []
+volumes:
+  - configMap
+  - downwardAPI
+  - emptyDir
+  - persistentVolumeClaim
+  - projected
+  - secret
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-kmm
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: scc-restricted-v2
+rules:
+  - apiGroups:
+      - security.openshift.io
+    resourceNames:
+      - restricted-v2
+    resources:
+      - securitycontextconstraints
+    verbs:
+      - use
+```
```

Comment:
Should we also add the respective `RoleBinding`, which bind the KMM operator's `ServiceAccount` with the `scc-restricted-v2` `ClusterRole`?

Reply:
Fixed, thanks!